test/formol/conditional.test.jsx

```diff
@@ -242,4 +242,31 @@ describe('Conditional field', () => {
         .props().readOnly
     ).toBeTruthy()
   })
+  it('initializes multiple fields on show condition change', async () => {
+    const wrapper = mount(
+      <Formol
+        item={{
+          switchMe: false,
+          lName: 'Doe',
+          fName: 'John',
+        }}
+      >
+        <Field name="switchMe" type="switch">
+          Switch me
+        </Field>
+        <Conditional show={({ switchMe }) => switchMe}>
+          <Field name="lName">Last name</Field>
+          <Field name="fName">First name</Field>
+        </Conditional>
+      </Formol>
+    )
+    const switchBtn = () => wrapper.find('input[name="switchMe"]')
+
+    await switchBtn().simulate('change', { target: { checked: true } })
+    const lName = () => wrapper.find('input[name="lName"]')
+    const fName = () => wrapper.find('input[name="fName"]')
+
+    expect(lName().props().value).toBe('Doe')
```

Comment:
Wouldn't be useful to test the values before changing the condition too ?

Reply:
Before changing the condition, they won't have any value at all as they are not mounted. That's why the `wrapper.find` comes after the switching to true, the Fields just don't exist at all..